rules/mandatory-object-comma.js

```diff
@@ -0,0 +1,117 @@
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'mandatory-object-comma';
+    }
+
+    test(data) {
+        const { formatMessage, addError } = this.outcome;
+
+        this.filterModulesData(data, 'twig').forEach(twigFileData => {
+            const {files, type, name} = twigFileData;
+            const {twig} = files;
+            const {content, path} = twig;
```

Comment:
Please add spaces between brackets

Reply:
Do we have the rules?

Reply:
Whole sniffer is written with spaces between brackets syntax.